kuma/wiki/tasks.py

```diff
@@ -37,6 +37,12 @@ def render_document(pk, cache_control, base_url):
     return document.rendered_errors
 
 
+@task()
+def clear_render_stale_document_lock():
+    lock = MemcacheLock('render-stale-documents-lock', expires=60 * 60)
```

Comment:
This is set multiple times. Perhaps use a constant? Also, it doesn't appear that you need it here since you just release it.